### PR TITLE
Docs: Initial docs

### PR DESCRIPTION
**Pull request by sam-finch-tezos**
# Context
Now that JsTz is coming along people might want to use it. We need to begin documenting what we're creating.
# Description
This PR creates some initial documentation and lays out some of the structure going forwards. 
# Manually testing the PR
[https://www.wikihow.com/Teach-Yourself-to-Read](www.wikihow.com/Teach-Yourself-to-Read)

### CODE REVIEW

File: docs/getting-started/installation/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Installing JsTz
+
+Installation is still quite manual at this stage. Plus most of us started with all the right things installed to begin with. 
+If anything doesn't work we'd love to hear from you. (Actually we'll hate it. We're only human and we're like fixing people's builds as much as anybody else, but we grudgingly accept it will make the product better so we'll at least try to feign enthusiasm).
```
Comment (asutosh-mourya) [inline docs/getting-started/installation/index.md:4]: while really funny,  I think such jokes could be easily misinterpreted by people who don't know the writer.

File: docs/getting-started/installation/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+``` sh
+git clone git@gitlab.com:tezos/tezos.git
+cd tezos
+git checkout eced5663816d95f314e44283eb0c058c63668a24
+nix-shell --run make
+```
+
+I think the view from core is that we don't actually need this step, just get the octez binaries from somewhere.
```
Comment (asutosh-mourya) [inline docs/getting-started/installation/index.md:23]: Can we please change this?

File: docs/getting-started/installation/index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Sep 29 17:00:17.809:   srs11afKWd1nKJymiEga83uuZ8ZAc39sUhaEkCk7mZXTb4tpKBgZDv at tick
+Sep 29 17:00:17.809:   2211000000000 with 3 messages
+Sep 29 17:00:17.811: Finished processing layer 1 head
+Sep 29 17:00:17.811:   BLwG6tnpms81kCtqddc1yhiFvESBgxLY5i9XyHSpUN5A82CvuqB at level 43 in 5.58ms
+Sep 29 17:00:17.811: Finished processing 1 layer 1 heads for levels 43 to 43
+Sep 29 17:00:17.811: [tz1gjaF81ZRRv: publish, add_messages, cement, refute] injection
+Sep 29 17:00:17.811:   Request pushed on 2023-09-29T16:00:17.811-00:00, treated in 1us, completed in 2us
+
+```
+If anything went wrong whe  the it's probably our fault not yours.
```
Comment (asutosh-mourya) [inline docs/getting-started/installation/index.md:71]: Please remove this :)

**Review by asutosh-mourya — CHANGES_REQUESTED** [review]
Left a few comments, I think written jokes could be easily misinterpreted by people who don't know the writer so it would be better to remove those.

File: docs/getting-started/my_first_smart_function/index.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+# Writing and Deploying a SmartFunction
+
+## Deploying your first contract
+
+
+### A very simple contract
+JsTz smart functions are modeled on http servers. These designed to be more familiar to most developers and to integrate better with the off chain part of your dApp.
```
Comment (alanmarkoTrilitech) [inline docs/getting-started/my_first_smart_function/index.md:7]: These designed -> These are designed

**Review by alanmarkoTrilitech — APPROVED** [review]
LGTM!

**Comment by johnyob** [comment]
Closing in favour of #35

**Pull request closed without merging**